src/promptflow-azure/promptflow/azure/operations/_async_run_uploader.py

```diff
@@ -172,12 +173,15 @@ async def _upload_snapshot(self) -> str:
         """Upload run snapshot to cloud."""
         logger.debug(f"Uploading snapshot for run {self.run.name!r}.")
         local_folder = self.run_output_path / LocalStorageFilenames.SNAPSHOT_FOLDER
+        # for some types of flex flow, even there is no flow.flex.yaml in the original flow folder,
```

Comment:
will this work for prompty? cc @lalala123123

Reply:
Skipped upload run for prompty run for now as offline synced.